docs/oid-info.md

```diff
@@ -23,35 +23,49 @@ The `.1` is added to the root OID for sigstore for all OIDs set by Fulcio.
 This contains the `issuer` claim from the OIDC Identity Token that was
 presented at the time the code signing certificate was requested to be created.
 This claim is the URI of the OIDC Identity Provider that digitally signed the
-identity token.
+identity token. For example: https://token.actions.githubusercontent.com.
 
-### 1.3.6.1.4.1.57264.1.2 | GitHub Workflow Trigger
+### 1.3.6.1.4.1.57264.3.1 | Source URI
 
-This contains the `event_name` claim from the GitHub OIDC Identity token that
-contains the name of the event that triggered the workflow run.
-[(docs)][github-oidc-doc]
+Should include a fully qualified source repository URL that the build was based
+on.
 
-### 1.3.6.1.4.1.57264.1.3 | GitHub Workflow SHA
+### 1.3.6.1.4.1.57264.3.2 | Source Digest
 
-This contains the `sha` claim from the GitHub OIDC Identity token that contains
-the commit SHA that the workflow run was based upon. [(docs)][github-oidc-doc]
+An immutable reference to a specific version of the source code that the build
+was based upon. Should include the digest type followed by the digest, e.g.
+`sha1:abc123`.
 
-### 1.3.6.1.4.1.57264.1.4 | GitHub Workflow Name
+### 1.3.6.1.4.1.57264.3.3 | Source Ref
 
-This contains the `workflow` claim from the GitHub OIDC Identity token that
-contains the name of the executed workflow. [(docs)][github-oidc-doc]
+The source ref that the build run was based upon. For example: `refs/head/main`.
 
-### 1.3.6.1.4.1.57264.1.5 | GitHub Workflow Repository
+### 1.3.6.1.4.1.57264.3.4 | Build Config URI
 
-This contains the `repository` claim from the GitHub OIDC Identity token that
-contains the repository that the workflow run was based upon.
-[(docs)][github-oidc-doc]
+A reference to the initiating build instructions.
 
-### 1.3.6.1.4.1.57264.1.6 | GitHub Workflow Ref
+### 1.3.6.1.4.1.57264.3.5 | Build Config Digest
 
-This contains the `ref` claim from the GitHub OIDC Identity token that contains
-the git ref that the workflow run was based upon.
-[(docs)][github-oidc-doc]
+An immutable reference to the specific version of the top-level build
+instructions. Should include the digest type followed by the digest, e.g.
+`sha1:abc123`.
+
+### 1.3.6.1.4.1.57264.3.6 | Build Siger URI
+
+Reference to specific build instructions that are responsible for signing. Can be the same as Build Config URI. For example a reusable workflow in GitHub Actions or a Circle CI Orbs.
+
+
+### 1.3.6.1.4.1.57264.3.7 | Build Siger Digest
```

Comment:
Could you please clarify this a bit similar to Build Siger URI? Is it up for a provider to decide what this is? E.g. SHA of an external GitHub Action or a hash of a bash script that did signing?

Reply:
Yes the intention here was to provide a immutable reference to the Build Signer URI. So in GitHub Actions case the URI would be the path with branch/tag ref of a reusable workflow. The digest would be the commit sha of the reusable workflow file. In GitHub's case the workflow ref usually contains a mutable branch/tag unless it's been referenced via a commit in the triggering workflow.

Hadn't thought about this directly referencing a bash script. I wonder if this is something that would be possible as all these values will be pulled from the CI/CD providers OIDC claims, so the CI provider would need to extract the location/digest of the script and place it in the token claims.

My intention with calling it the "build signer" was to make it clear that it's the job/container/thing where the CI/CD's OIDC token was requested. It's technically possible to pass this token back to another job where the signing happens though.

In GitHub actions case "Build Signer" and "Build Config" are identical if you're **not** using a reusable workflow. I imagine this would also be the case for Circle CI if using Orbs or not.

Another name idea for "Build Signer" could be "Build Identity" but potentially more confusing.